Classes/Queue/DoctrineQueue.php

```diff
@@ -191,15 +200,38 @@ protected function reserveMessage(?int $timeout = null): ?Message
         }
         $this->reconnectDatabaseConnection();
 
+        $qb = $this->connection->createQueryBuilder();
+        $qb
+            ->select('*')
+            ->from($this->connection->quoteIdentifier($this->tableName))
+            ->where(
+                $qb->expr()->and(
+                    $qb->expr()->eq('state', $qb->expr()->literal('ready')),
```

Comment:
I'm wondering about the many literal comparisons.
Why not
```php
->andWhere('state = :state')->setParameter('state', 'ready');
```
?

Reply:
Good point. I'm doing too little "direct DB stuff"… 😬